includes/class-cron.php

```
@@ -136,33 +136,36 @@ private function schedule_page_cron_jobs(): void {
 			return;
 		}
 
-		$options      = get_option( 'wppo_settings', array() );
-		$exclude_urls = Util::process_urls( $options['preload_settings']['excludePreloadCache'] ?? array() );
+		$options          = get_option( 'wppo_settings', array() );
+		$raw_exclude_urls = Util::process_urls( $options['preload_settings']['excludePreloadCache'] ?? array() );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'PerformanceOptimise\Inc\Util' in method 'schedule_page_cron_jobs'. (undefined)

(StaticAccess)